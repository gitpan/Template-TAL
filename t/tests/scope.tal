<html xmlns:tal="http://xml.zope.org/namespaces/tal">

  <div tal:define="foo string:bar">

    <div tal:define="global baz string:ning" tal:omit-tag="string:1" />
  
    <p tal:content="foo">should be bar</p>
    <p tal:content="baz">should be ning</p>
  
    <div tal:define="foo string:baz">
      <div tal:define="global baz string:wop" tal:omit-tag="string:1" />
  
      <p tal:content="foo">should be baz</p>
      <p tal:content="baz">should be wop</p>
    </div>
  
    <p tal:content="foo">should be bar</p>
    <p tal:content="baz">should be wop</p>

  </div>
  
</html>
