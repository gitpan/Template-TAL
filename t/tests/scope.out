<html>
<div>
  <p>bar</p>
  <p>ning</p>
  <div>
    <p>baz</p>
    <p>wop</p>
  </div>
  <p>bar</p>
  <p>wop</p>
</div>
</html>
